src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -217,14 +218,38 @@ private static SwerveModule createSwerveModule(
   }
 
   public Rotation2d getGyroscopeRotation() {
-    return pigeon.getRotation2d();
+    return gyro.getRotation2d();
     // We have to invert the angle of the NavX so that rotating the robot counter-clockwise makes
     // the angle increase.
     // return Rotation2d.fromDegrees(360.0 - navx.getYaw());
   }
 
   public void setGyroscopeRotation(double angleDeg) {
-    pigeon.setYaw(angleDeg);
+    gyro.setYaw(angleDeg);
```

Comment:
Suggest the drivetrain should return the WrappedGryo object, rather than adding each of these functions.